### ISSUE TITLE
Cinnamon crashes when opening new windows (especially Firefox)

### ISSUE THREAD

**Issue by daviewales** [linuxmint/linuxmint#719]
I'm getting repeated Cinnamon crashes when opening new windows in Firefox.
(The fallback mode is not affected.)

For example, see the following journalctl log after opening a new Firefox window. (New tabs are safe.)

<details>

```
Sep 16 12:01:23 dwales-thinkpad kernel: cinnamon[50966]: segfault at 20 ip 0000794aab79420b sp 00007ffc248a7360 error 4 in libmuffin-clutter-0.so.0.0.0[794aab776000+a3000] likely on CPU 2 (core 2, socket 0)
Sep 16 12:01:23 dwales-thinkpad kernel: Code: 00 49 8b 47 20 80 b8 20 03 00 00 1f 76 38 80 88 20 03 00 00 e0 41 f6 47 18 02 74 07 80 88 23 03 00 00 10 48 8b b8 48 01 00 00 <48> 8b 47 20 f6 47 1c 01 75 12 0f b6 80 20 03 00 00 f7 d0 a8 e0 74
Sep 16 12:01:23 dwales-thinkpad systemd[6318]: Started vte-spawn-7348d349-950b-4515-b384-ac1cccb2ef3a.scope - VTE child process 54901 launched by gnome-terminal-server process 54859.
Sep 16 12:01:23 dwales-thinkpad systemd[1]: Started systemd-coredump@55-54908-0.service - Process Core Dump (PID 54908/UID 0).
Sep 16 12:01:25 dwales-thinkpad systemd-coredump[54910]: Process 50966 (cinnamon) of user 1000 dumped core.
                                                         
                                                         Module libzstd.so.1 from deb libzstd-1.5.5+dfsg2-2build1.1.amd64
                                                         Module libsystemd.so.0 from deb systemd-255.4-1ubuntu8.4.amd64
                                                         Module libudev.so.1 from deb systemd-255.4-1ubuntu8.4.amd64
                                                         Stack trace of thread 50966:
                                                         #0  0x0000794aab79420b _clutter_actor_queue_only_relayout (libmuffin-clutter-0.so.0 + 0x4b20b)
                                                         #1  0x0000794aab199239 st_bin_set_child (libst.so + 0x22239)
                                                         #2  0x0000794aa9fb7b16 n/a (libffi.so.8 + 0x7b16)
                                                         #3  0x0000794aa9fb43ef n/a (libffi.so.8 + 0x43ef)
                                                         #4  0x0000794aa9fb70be ffi_call (libffi.so.8 + 0x70be)
                                                         #5  0x0000794aab064aa3 _ZN3Gjs8Function6invokeEP9JSContextRKN2JS8CallArgsENS3_6HandleIP8JSObjectEEP11_GIArgument (libcjs.so.0 + 0x58aa3)
                                                         #6  0x0000794aab065389 _ZN3Gjs8Function4callEP9JSContextjPN2JS5ValueE (libcjs.so.0 + 0x59389)
                                                         #7  0x0000794aa7b622a0 n/a (libmozjs-115.so.0 + 0x1622a0)
                                                         #8  0x0000794aa80182ae n/a (libmozjs-115.so.0 + 0x6182ae)
                                                         #9  0x000016a2977efe5c n/a (n/a + 0x0)
                                                         #10 0x000016a2977f5986 n/a (n/a + 0x0)
                                                         #11 0x000016a29783f9c4 n/a (n/a + 0x0)
                                                         #12 0x000016a2977ed4e9 n/a (n/a + 0x0)
                                                         #13 0x0000794aa81cf3e6 n/a (libmozjs-115.so.0 + 0x7cf3e6)
                                                         #14 0x0000794aa7b5e17e n/a (libmozjs-115.so.0 + 0x15e17e)
                                                         #15 0x0000794aa7b61d6c n/a (libmozjs-115.so.0 + 0x161d6c)
                                                         #16 0x0000794aa7b62177 n/a (libmozjs-115.so.0 + 0x162177)
                                                         #17 0x0000794aa7b626e9 n/a (libmozjs-115.so.0 + 0x1626e9)
                                                         #18 0x0000794aa7b70409 n/a (libmozjs-115.so.0 + 0x170409)
                                                         #19 0x0000794aa7b567e7 n/a (libmozjs-115.so.0 + 0x1567e7)
                                                         #20 0x0000794aa7b61e0b n/a (libmozjs-115.so.0 + 0x161e0b)
                                                         #21 0x0000794aa7b62177 n/a (libmozjs-115.so.0 + 0x162177)
                                                         #22 0x0000794aa7b626e9 n/a (libmozjs-115.so.0 + 0x1626e9)
                                                         #23 0x0000794aa7be11c9 _Z20JS_CallFunctionValueP9JSContextN2JS6HandleIP8JSObjectEENS2_INS1_5ValueEEERKNS1_16HandleValueArrayENS1_13MutableHandleIS6_EE (libmozjs-115.so.0 + 0x1e11c9)
                                                         #24 0x0000794aab05ca65 Call (libcjs.so.0 + 0x50a65)
                                                         #25 0x0000794aab097a81 _ZN3Gjs7Closure7marshalEP7_GValuejPKS1_PvS5_ (libcjs.so.0 + 0x8ba81)
                                                         #26 0x0000794aab9c72fa g_closure_invoke (libgobject-2.0.so.0 + 0x152fa)
                                                         #27 0x0000794aab9f690c n/a (libgobject-2.0.so.0 + 0x4490c)
                                                         #28 0x0000794aab9e7591 n/a (libgobject-2.0.so.0 + 0x35591)
                                                         #29 0x0000794aab9e7a98 g_signal_emit_by_name (libgobject-2.0.so.0 + 0x35a98)
                                                         #30 0x0000794aab50752d _meta_window_shared_new (libmuffin.so.0 + 0xc152d)
                                                         #31 0x0000794aab53346b meta_window_x11_new (libmuffin.so.0 + 0xed46b)
                                                         #32 0x0000794aab51bd40 handle_other_xevent (libmuffin.so.0 + 0xd5d40)
                                                         #33 0x0000794aab51dca5 meta_x11_display_handle_xevent (libmuffin.so.0 + 0xd7ca5)
                                                         #34 0x0000794aaadd99c0 n/a (libgdk-3.so.0 + 0x8b9c0)
                                                         #35 0x0000794aaade185e n/a (libgdk-3.so.0 + 0x9385e)
                                                         #36 0x0000794aaad856eb gdk_display_get_event (libgdk-3.so.0 + 0x376eb)
                                                         #37 0x0000794aaade1c5e n/a (libgdk-3.so.0 + 0x93c5e)
                                                         #38 0x0000794aab8c65b5 n/a (libglib-2.0.so.0 + 0x5d5b5)
                                                         #39 0x0000794aab925717 n/a (libglib-2.0.so.0 + 0xbc717)
                                                         #40 0x0000794aab8c6f77 g_main_loop_run (libglib-2.0.so.0 + 0x5df77)
                                                         #41 0x0000794aab4fcc3f meta_run (libmuffin.so.0 + 0xb6c3f)
                                                         #42 0x0000570d44504b0c main (cinnamon + 0x2b0c)
                                                         #43 0x0000794aab22a1ca __libc_start_call_main (libc.so.6 + 0x2a1ca)
                                                         #44 0x0000794aab22a28b __libc_start_main_impl (libc.so.6 + 0x2a28b)
                                                         #45 0x0000570d44504cb5 _start (cinnamon + 0x2cb5)
                                                         
                                                         Stack trace of thread 50975:
                                                         #0  0x0000794aab31b4cd __GI___poll (libc.so.6 + 0x11b4cd)
                                                         #1  0x0000794aab92566e n/a (libglib-2.0.so.0 + 0xbc66e)
                                                         #2  0x0000794aab8c5a53 g_main_context_iteration (libglib-2.0.so.0 + 0x5ca53)
                                                         #3  0x0000794aabc2a595 n/a (libdconfsettings.so + 0x7595)
                                                         #4  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50974:
                                                         #0  0x0000794aab31b4cd __GI___poll (libc.so.6 + 0x11b4cd)
                                                         #1  0x0000794aab92566e n/a (libglib-2.0.so.0 + 0xbc66e)
                                                         #2  0x0000794aab8c6f77 g_main_loop_run (libglib-2.0.so.0 + 0x5df77)
                                                         #3  0x0000794aabb2f422 n/a (libgio-2.0.so.0 + 0x11a422)
                                                         #4  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51009:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51007:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51008:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 54339:
                                                         #0  0x0000794aab31b4cd __GI___poll (libc.so.6 + 0x11b4cd)
                                                         #1  0x0000794a9482a566 n/a (libpulse.so.0 + 0x34566)
                                                         #2  0x0000794a948136c4 pa_mainloop_poll (libpulse.so.0 + 0x1d6c4)
                                                         #3  0x0000794a9481e193 pa_mainloop_iterate (libpulse.so.0 + 0x28193)
                                                         #4  0x0000794a9481e258 pa_mainloop_run (libpulse.so.0 + 0x28258)
                                                         #5  0x0000794a9482e771 n/a (libpulse.so.0 + 0x38771)
                                                         #6  0x0000794a947cd3eb n/a (libpulsecommon-16.1.so + 0x5b3eb)
                                                         #7  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #8  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51010:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51014:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51011:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50982:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794a95f1d7ad n/a (iris_dri.so + 0x11d7ad)
                                                         #3  0x0000794a95efca8b n/a (iris_dri.so + 0xfca8b)
                                                         #4  0x0000794a95f1d6dc n/a (iris_dri.so + 0x11d6dc)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50971:
                                                         #0  0x0000794aab31b4cd __GI___poll (libc.so.6 + 0x11b4cd)
                                                         #1  0x0000794aab92566e n/a (libglib-2.0.so.0 + 0xbc66e)
                                                         #2  0x0000794aab8c5a53 g_main_context_iteration (libglib-2.0.so.0 + 0x5ca53)
                                                         #3  0x0000794aab8c5aa9 n/a (libglib-2.0.so.0 + 0x5caa9)
                                                         #4  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51055:
                                                         #0  0x0000794aab32725d syscall (libc.so.6 + 0x12725d)
                                                         #1  0x0000794aab91f40d g_cond_wait (libglib-2.0.so.0 + 0xb640d)
                                                         #2  0x0000794aab89052b n/a (libglib-2.0.so.0 + 0x2752b)
                                                         #3  0x0000794aab89058c g_async_queue_pop (libglib-2.0.so.0 + 0x2758c)
                                                         #4  0x0000794aa9d380fb n/a (libpangoft2-1.0.so.0 + 0xc0fb)
                                                         #5  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51025:
                                                         #0  0x0000794aab32725d syscall (libc.so.6 + 0x12725d)
                                                         #1  0x0000794aab91f40d g_cond_wait (libglib-2.0.so.0 + 0xb640d)
                                                         #2  0x0000794aab89052b n/a (libglib-2.0.so.0 + 0x2752b)
                                                         #3  0x0000794aab89058c g_async_queue_pop (libglib-2.0.so.0 + 0x2758c)
                                                         #4  0x0000794aa9d380fb n/a (libpangoft2-1.0.so.0 + 0xc0fb)
                                                         #5  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50986:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794a95f1d7ad n/a (iris_dri.so + 0x11d7ad)
                                                         #3  0x0000794a95efca8b n/a (iris_dri.so + 0xfca8b)
                                                         #4  0x0000794a95f1d6dc n/a (iris_dri.so + 0x11d6dc)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50985:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794a95f1d7ad n/a (iris_dri.so + 0x11d7ad)
                                                         #3  0x0000794a95efca8b n/a (iris_dri.so + 0xfca8b)
                                                         #4  0x0000794a95f1d6dc n/a (iris_dri.so + 0x11d6dc)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50972:
                                                         #0  0x0000794aab32725d syscall (libc.so.6 + 0x12725d)
                                                         #1  0x0000794aab91f40d g_cond_wait (libglib-2.0.so.0 + 0xb640d)
                                                         #2  0x0000794aab89052b n/a (libglib-2.0.so.0 + 0x2752b)
                                                         #3  0x0000794aab8f8043 n/a (libglib-2.0.so.0 + 0x8f043)
                                                         #4  0x0000794aab8f4c82 n/a (libglib-2.0.so.0 + 0x8bc82)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51012:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 51013:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794aa8334cb9 _ZN7mozilla6detail21ConditionVariableImpl4waitERNS0_9MutexImplE (libmozjs-115.so.0 + 0x934cb9)
                                                         #3  0x0000794aa8336e15 _ZN7mozilla6detail21ConditionVariableImpl8wait_forERNS0_9MutexImplERKNS_16BaseTimeDurationINS_27TimeDurationValueCalculatorEEE (libmozjs-115.so.0 + 0x936e15)
                                                         #4  0x0000794aa7c1cf5e n/a (libmozjs-115.so.0 + 0x21cf5e)
                                                         #5  0x0000794aa7c1ce47 n/a (libmozjs-115.so.0 + 0x21ce47)
                                                         #6  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #7  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         
                                                         Stack trace of thread 50981:
                                                         #0  0x0000794aab298d61 __futex_abstimed_wait_common64 (libc.so.6 + 0x98d61)
                                                         #1  0x0000794aab29b7dd __pthread_cond_wait_common (libc.so.6 + 0x9b7dd)
                                                         #2  0x0000794a95f1d7ad n/a (iris_dri.so + 0x11d7ad)
                                                         #3  0x0000794a95efca8b n/a (iris_dri.so + 0xfca8b)
                                                         #4  0x0000794a95f1d6dc n/a (iris_dri.so + 0x11d6dc)
                                                         #5  0x0000794aab29ca94 start_thread (libc.so.6 + 0x9ca94)
                                                         #6  0x0000794aab329c3c __clone3 (libc.so.6 + 0x129c3c)
                                                         ELF object binary architecture: AMD x86-64
Sep 16 12:01:25 dwales-thinkpad systemd[1]: systemd-coredump@55-54908-0.service: Deactivated successfully.
Sep 16 12:01:25 dwales-thinkpad systemd[1]: systemd-coredump@55-54908-0.service: Consumed 1.164s CPU time.
```

</details>

I have tried `Melange > Reset Cinnamon Settings`. But I'm still getting the crashes.
_Sometimes_ other applications will trigger a crash (I've had it with Thunderbird and Gnome Terminal) , but it's not as reproducible as with Firefox.

I just tried rebooting, and it crashed as soon as I opened Firefox.

Logging out, then back in with Cinnamon (Software Rendering) did _not_ crash.

Logging out again, then back in with Cinnamon (Default) crashed immediately when I opened Firefox.

Logging out again, then back in with Cinnamon (Wayland experimental) did _not_ crash.

Logging out again, then back in with XFCE did _not_ crash. (NOTE: The crash was occurring in Cinnamon before I installed XFCE. I installed it specifically to test this issue. So I know that installing XFCE did not cause the crash.)

So in summary, the issue appears to be with Cinnamon default hardware acceleration and Firefox.

**Comment by daviewales** [comment]
I've just tried Cinnamon  (Default) again today, and it's not crashing... I've recently installed the Firefox 130.0.1 update, so perhaps that's fixed it? Will report back if it starts crashing again.

**Comment by JosephMcc** [comment]
The problem with this is reproducing it. I use Firefox everyday as do most Mint users. You are the only one to report this and we'd hear about it a lot more if it was very widespread. Since it seems to be something somewhat unique to your setup, it's pretty much impossible to troubleshoot.

**Comment by daviewales** [comment]
Thanks, I appreciate that. It's mysteriously stopped crashing today, but it was very reliably reproducing for me last week, even after reboots. I'll close this for now, but will reopen if it happens again, and will try to find more information for troubleshooting.
